client/components/deposits-overview/utils.ts

```diff
@@ -7,7 +7,8 @@ type NextDepositTableData = {
 	id?: string;
 	date: number;
 	status: string;
-	amount: string;
+	formattedAmount: string;
+	amount: number;
```

Comment:
We might not need this any more – will need to remember to revert when everything else is working correctly.

Reply:
Yeah this isn't needed anymore. I've reverted it in 83a9afa615114f2a3183217e1462e002963df5ca